addons/hr_holidays/tests/test_hr_holidays_tour.py

```diff
@@ -38,6 +38,5 @@ def test_hr_holidays_tour(self):
             'date_from': '2022-01-01',
             'date_to': '2022-12-31',
         })
-        allocation.action_validate()
```

Comment:
Because the leave type created above does not define `allocation_validation_type`, which is set default to "no". With that, the allocation is automatically validated. Not sure how the test passed in the first place.